webextensions/api/userScripts.json

```
@@ -67,37 +198,83 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/register",
             "support": {
               "chrome": {
+                "version_added": "115"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "resetWorldConfiguration": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "115"
```

```suggestion
                "version_added": "133"
```

Note: not 120 but 133, connected to the "multiple user script worlds feature":
- Commit: https://chromium.googlesource.com/chromium/src/+/78e7de28d840e7455b456bf7a7c2c3e6c25e71e4
- Version: https://chromium.googlesource.com/chromium/src/+/78e7de28d840e7455b456bf7a7c2c3e6c25e71e4/chrome/VERSION
- Specification: https://github.com/w3c/webextensions/blob/main/proposals/multiple_user_script_worlds.md